src/components/ResourceBrowser/ResourceBrowser.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useMemo } from 'react'
+import { NavLink, useHistory } from 'react-router-dom'
+import { showError, getUserRole, DevtronProgressing, useAsync } from '@devtron-labs/devtron-fe-common-lib'
+import PageHeader from '../common/header/PageHeader'
+import { sortObjectArrayAlphabetically } from '../common'
+import { ALL_NAMESPACE_OPTION, K8S_EMPTY_GROUP, SIDEBAR_KEYS } from './Constants'
+import { URLS } from '../../config'
+import ClusterSelectionList from '../ClusterNodes/ClusterSelectionList'
+import { getClusterList, getClusterListMin } from '../ClusterNodes/clusterNodes.service'
+import { ClusterDetail } from '../ClusterNodes/types'
+import { ReactComponent as Add } from '../../assets/icons/ic-add.svg'
+import './ResourceBrowser.scss'
+
+const addClusterButton = () => (
+    /* funcs returning JSX should be memoized if declared inside react func body
```

Comment:
Remove this comment, as it is not making sense.
And convert it Functional Component.